### ISSUE TITLE
Livechat: Don't expose the agent's name

### ISSUE THREAD

**Issue by mrsimpson** [RocketChat/Rocket.Chat#15237]
## Motivation

As a livechat-operator, I don't want my agent's names to be exposed to the customer. 

## What's the issue?

While the email-address can be configured as hidden, the name appears in the header, in the messages and in the "is typing" notification.

*Remark: While hiding the name from livechat visitors, agents shall of course be able to address their colleagues using their real names. Thus, changing all "real names" of the users to a common one is no option.*

## Implementation alternatives

- Provide an alias which can be configured *for each agent*
- Provide a common alias for all agents (may of the department, if configured)
- Hide the name from all UI-places

Since livechat is supposed to create a human, direct channel, the ability to have an explicit name for each agent (which could of course be harmonized for all agents manually) is preferred.

**Comment by mrsimpson** [comment]
@renatobecker wdyt? Do you have something like this on your list for the new livechat widget? If not: How can we contribute?

**Comment by renatobecker** [comment]
> @renatobecker wdyt? Do you have something like this on your list for the new livechat widget? If not: How can we contribute?

I think you're looking for this -> https://github.com/RocketChat/Rocket.Chat.Livechat/pull/279

**Comment by mrsimpson** [comment]
@renatobecker yup, that’s the implementation of the third option I mentioned 👍
Wdyt about an alias? It seems to be common that agents operate under an alias

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by renatobecker** [comment]
> @renatobecker yup, that’s the implementation of the third option I mentioned 👍
> Wdyt about an alias? It seems to be common that agents operate under an alias
> 
> Sent with [GitHawk](http://githawk.com)

Yeah, the idea about providing an `alias` for Livechat Agents is something already planned, it's on our roadmap and it will be released in the short-term.

**Comment by KevLehman** [comment]
This is a setting now, thx!
